src/stylesheets/_responsiveness.scss

```diff
@@ -1,6 +1,6 @@
 @mixin respond($breakpoint) {
   @if ($breakpoint == smallest) {
-    @media (max-width: 422px) {
```

Comment:
I'd stray away from changing these constants. Please, if any, try and adjust the component using a `min-width` or a `max-width`.

Reply:
I mentioned this above, but I changed this to accommodate for smaller phone resolutions, such as the iPhone X - I can change this back if need be.

I'm also confused about what you mean by your second comment. I've checked the responsive of the component itself, and I think it looks fine. Let me know if there's something specific (per screen resolution) you want me to take a look at!

(In short, `max-width` on `EmailSubscription` can work, but it might not be necessary.)

Reply:
When you import the component here: 
https://github.com/CruzHacks/cruzhacks-2021-website/blob/9b61e7d46b04f92d808f3bea3949577917232fb8/src/views/Hero/index.view.tsx#L27-L32
You should wrap this within its own container and style that to fit within the larger view.

Reply:
Little confused about the difference, but I implemented it regardless! Let me know if we should change back the smallest resolution also, but take a look at it!

Reply:
Because remember that each component is styled by itself in the `components` directory, but we need to integrate the component to each view and that's where we fix widths and all that good stuff.

Reply:
As to your earlier point about changing the dimension of this, then I think you should be ok! I'm not sure how we came up with the breakpoints last year so this change is ok!